changelog/README.rst

```diff
@@ -0,0 +1,39 @@
+*********************
+Manging the Changelog
+*********************
+
+This directory contains "newsfragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next ``CHANGELOG``.
+
+The ``CHANGELOG`` will be read by users, so this description should be aimed to
+molecule users instead of describing internal changes which are only relevant
+to the developers.
+
+Make sure to use full sentences with correct case and punctuation, for
+example::
+
+    Fix issue with non-ascii messages from the ``warnings`` module.
```

Comment:
This example doesn't give the reader an understanding of what the issue was. It's kinda blurry. I think it might be better to re-phrase it as:
```suggestion
    Fix broken output of non-ascii messages produced by the ``warnings`` module.
```

In some projects (tox?) they also encourage tagging authors of the change in their template.

Reply:
I'd also like @acozine about the best wording as it's docs-related.